repl/variables.go

```diff
@@ -0,0 +1,38 @@
+package repl
+
+import (
+	"os"
+	"strings"
+
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+func setVariables(values map[string]string) error {
+	for key, value := range values {
+		err := os.Setenv(key, value)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func echo(input string) {
+	variables := strings.Split(input, " ")
+	for _, v := range variables {
+		fields := strings.SplitN(v, "=", 2)
+		if len(fields) > 0 {
+			var value string
+			key := strings.TrimSpace(fields[0])
+			if len(fields) > 1 {
+				value = strings.TrimSpace(fields[1])
+			}
+
+			err := os.Setenv(key, value)
```

Comment:
Setting variables shouldn't happen from the echo command. Instead you should create a function that translate variables to their values (recognize variables starting with `$`. Then every command could use ENV variables.
Setting a variable should be another command (`set`/`export`) and should be contained inside the node software.

Reply:
sorry if this is a silly question but what is ENV variable?

Reply:
Environment variables are variables variables KEY/VALUE pairs that are exposed widely in operating systems so software can read and use them.
We want to simulate this feature inside the spacemesh node without affecting the actual operating system environment variables.
The repl should be able read such variables from the node. (As NetworkID and DataFilePath for example)

Reply:
so let's see if I understood right. All that I have to do is get the variables that the user puts on console (e.g. $NetworkID), ask the node the variable and print it on the console, right? there is no need of a method like `setVariables` like I created.

Reply:
For now you can start with this. in the future we might want to enable users to set variables through the console aswell.